src/generic_ui/locales/en/messages.json

```diff
@@ -175,14 +175,30 @@
     "description": "Message helping make sure the user understands that personally identifiable information may be included in their feedback to uProxy.",
     "message": "Your feedback and email address will be sent to uProxy.org. Your logs, network information and email may include personally identifiable information."
   },
-  "THANK_YOU": {
+  "FEEDBACK_TITLE": {
     "description": "",
     "message": "Thank you!"
   },
+  "FEEDBACK_TITLE_4": {
+    "description": "",
+    "message": "Trouble signing in"
+  },
+  "FEEDBACK_TITLE_5": {
+    "description": "",
+    "message": "No friends to proxy from"
+  },
   "FEEDBACK_SUBMITTED": {
     "description": "User feedback was successfully submitted.",
     "message": "Your feedback has been submitted to the uProxy development team."
   },
+  "FEEDBACK_SUBMITTED_4": {
+    "description": "User feedback was successfully submitted.",
+    "message": "Login will fail if user's are trying a social network that's banned in their region. Click <uproxy-faq-link anchor='willUproxyWork'>here</uproxy-faq-link> for additional help."
```

Comment:
s/user's/users

Would this message read better in second person rather than third? ("Login will fail if you choose a social network that's blocked..."